int/config/network.go

```diff
@@ -167,21 +174,21 @@ func (n *NetworkManager) SwitchNetwork(selectedNetworkStr string) error {
 	return nil
 }
 
-//nolint:typecheck,nolintlint
-//go:embed config_network.yaml
-var configData embed.FS
```

Comment:
Shouldn't we embed a default yaml file ? 
That would avoid us having to remake it one from the code and make it simpler to modify since we would easily know where the yaml file is in the code.

Reply:
the file is created if not found

Reply:
well, I'm not convinced because the code will create a file if it doesn't exists

Reply:
As discussed during our daily, we might want to add comments in the file to warn users that the default nodes (Massa Mainnet and Buildnet) shouldn't be modified by users if they don't want to break MS.
Also, we might want to add a short comment at the beginning explaining what this config file is and other stuff like that

I'm unsure it is possible to add comments from code (at least using yaml marshall), so maybe that would force us to have a default yaml file embedded

Reply:
@SlnPons does the doc is sufficient or do we need to do what Thomas suggest (adding comments in the .yaml file)?